highspeed_linear: 2.0
highspeed_throttle: 1.0
highspeed_angular: 1.0

lowspeed_linear: 1.0
lowspeed_throttle: 0.5
lowspeed_angular: 0.5